features/unit/abijson.feature

```diff
@@ -89,9 +89,9 @@ Feature: AbiJson
 
 
   Scenario Outline: Get a method by name from Interface
-    When I create the Method object from method signature "<methodsig>"
-    And I create another Method object from method signature "<extramethod>"
-    And I create an Interface object from the Method object
+    When I append to my Method objects list in the case of a non-empty signature "<methodsig>"
```

Comment:
Thinking more about it, you might be able to reuse the old instruction `I create the Method object from method signature...` if it also appends to the list... but for clarity and easier backwards compatibility, this new step is probably preferred.